apps/console/src/features/identity-providers/data/identity-provider-templates/templates/apple/create-wizard-help.tsx

```diff
@@ -0,0 +1,241 @@
+    }, [ identityProviderConfig ]);
+    
+    return (
+        <div data-componentid={ componentId }>
+            <Message
+                type="info"
+                header={
+                    t("console:develop.features.authenticationProvider.templates.apple." +
+                    "wizardHelp.preRequisites.heading")
+                }
+                content={
+                    (<>
+                        <p>
+                            <Trans
+                                i18nKey={
+                                    "console:develop.features.authenticationProvider.templates.apple." +
+                                    "wizardHelp.preRequisites.getCredentials"
+                                }
+                            >
+                                Before you begin, create a <strong>Sign in With Apple Enabled App</strong> on <a
+                                    href="https://developer.apple.com/"
+                                    target="_blank"
+                                    rel="noopener noreferrer">
+                                Apple Developer Program
+                                </a> with a <strong>Services ID</strong> and a <strong>Private Key</strong>.
+                            </Trans>
+                        </p>
+                        <p>
+                            <Trans
+                                i18nKey={
+                                    "console:develop.features.authenticationProvider.templates.apple." +
+                                    "wizardHelp.preRequisites.configureWebDomain"
+                                }
+                            >
+                            Use the following as a <strong>Web Domain</strong>.
+                            </Trans>
+                            <CopyInputField
+                                className="copy-input-dark spaced"
+                                value={ new URL(config?.deployment?.serverOrigin)?.hostname }
+                            />
+                        </p>
+                        <p>
+                            <Trans
+                                i18nKey={
+                                    "console:develop.features.authenticationProvider.templates.apple." +
+                                    "wizardHelp.preRequisites.configureReturnURL"
+                                }
+                            >
+                            Add the following URL as a <strong>Return URL</strong>.
+                            </Trans>
+                            <CopyInputField
+                                className="copy-input-dark spaced"
+                                value={ config?.deployment?.customServerHost + "/commonauth" }
+                            />
+                            <a
+                                href={
+                                    "https://developer.apple.com/documentation/sign_in_with_apple" + 
```

Comment:
Shall we add this to constants?

Reply:
Since this is authenticator specific file within `data` directory and we don't have such variables added to constants file, wouldn't be better to have it here?